jouko-api/src/main/java/fi/metatavu/jouko/api/AdminApiImpl.java

```diff
@@ -266,10 +287,15 @@ public Response deleteInterruption(Long groupId) throws Exception {
 
   @Override
   public Response deleteControllerDevice(Long controllerDeviceId) throws Exception {
-    // TODO Auto-generated method stub
-    System.out.println("Granny's leg");
-    System.out.println("Leg of the granny");
-    return null;
+    deviceController.deleteControllerDevice(controllerDeviceId);
+    return Response.ok().build();
   }
-  
+
+  // @Override
```

Comment:
@apachejuice You might want to look at this, all the API paths need to be in the AdminApiImpl file and uses a funky @Override as well to work but for this particular method I tried to create by replicating existing methods, it didn't like the way I did it.

We would pass the keycloakId to find the user to delete and the token for authentication that some 3rd party wouldn't just be able to delete the user without proper authentication.

Reply:
This is gonna be needed for a Java test as well to delete the user.